### ISSUE TITLE
possible rain-ndvi TS length miss-match

### ISSUE THREAD

**Issue by robinkraft** [wri/forma-clj#3]
TL;DR: rain ts likely to be 1-value shorter than NDVI. Can't ignore this, so we should just duplicate the previous value for the current month if there's no data. The implication is that we would also have to rerun FORMA for t and t-1.

This may be mostly a non-issue, but a rain ts and a 16-day MODIS ts will probably have different lengths. Dan says rain isn't hugely important, but his functions do assume the timeseries are the same length. But if there's no data yet for the most recent month, even ts-expander can't handle that.

For example, if we're running FORMA through the Jan. 17 period (which spills into February to incude 2/1), the rain value for February won't be available until March. So there won't be an expanded ts until sometime in early March, when the rain file will be updated. So the rain ts will be one element shorter than the ndvi.

So we could either not estimate for the most recent period, or we could just duplicate the previous value - we're only looking for broad trends of rainfall to filter out droughts and the like.

The (possibly) complicating factor is that if we just duplicate the previous value, eventually that duplicated value will be duplicated again as needed to make the ts the right length. This would give us a timeseries with the same value at the end, staying the same indefinitely. Eventually rain would become useless.

So, it may be that each FORMA run will also have to be for the now-2nd most recent month, so that once we have good data for the 2nd most recent month, we can update the probabilities for that period.
